Jenkinsfile

```diff
@@ -307,6 +377,80 @@ pipeline {
                         }
                     }
                 }
+                stage('z-master') {
```

Comment:
left that as an earlier comment (but perhaps it got buried; consider renaming these to;

- `z-integration`
- `z-integration-cli`

(and rename the bundles accordingly); it describes better what they are, and make it easier to find the right `bundle.tar.gz` if you're looking for a specific test failure

Reply:
done